Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/TextToolIntegrationTest.java

```diff
@@ -145,6 +143,23 @@ public void testDialogKeyboardTextBoxAppearanceOnStartup() {
 		checkTextBoxDimensionsAndDefaultPosition();
 	}
 
+	@Test
+	public void testConfirmButton() {
```

Comment:
Please describe what the test is actually testing and in which situation, as the behaviour is situation dependent.